r_scripts/analysis/oregon_part_a.Rmd

```diff
@@ -0,0 +1,1083 @@
+
+The **cardiopulmonary health outcomes** are all respiratory, asthma, COPD, cardiovascular disease, heart failure, and ischemic heart disease.
+
+As for **exposure methods**, we use Geo-Weighted Regression and subtracted off the 'Background' estimates of smoke, which I believe are the monthly averages of PM~2.5~ for a given grid. However, the 'Background' is the mean of total period from May 1st to Sep 30th 2013 so I have to assign the mean for every day. Then use the 10 units of the smoke for analysis.
+
+The **analytic method** is the conditional logistic regression model using the *survival* package in R. This regression is controlling the confounders. Each conditional logistic regression model accounts for the subject, and adjusts for temperature from the WRF-Chem model. The conditional logistic regression model \beta represents the change in risk of an event associated with a short-term 10 units' increase in exposure, and can be calculated as an average difference between exposure at the index time and a weighted average of exposure at all times.
+
+
+**Library:**  
+tidyverse is for data cleaning, also includes ggplot and readr.  
+data.table is for quick reading files.  
+htmlTable is for writing table.  
+survival is for running conditional logistic regression model.
+
+```{r library, include=FALSE, echo=FALSE, message=FALSE, warning=FALSE}
+library(tidyverse)
+library(data.table)
+library(survival)
+library(htmlTable)
+```
+
+
+
+**Time-Stratified Case-Crossover Data Set (same day of week within wildfire season May - September)**
+
+The data set is from the case crossover data set, which is made before. To save the running time, the data set is wildfire season and people's first visit for same date. Then filter the Emergency postal of service and non-missing gender visits.
+
+```{r ER data making, include = F, eval = FALSE}
+### Limit to only ER visits
+## pos = 23
+var_list <- c('respiratory', 'asthma', 'pneumonia',  'acute_bronch', 'copd', 

```

Comment:
Cut out the outcomes we don't use for all the code below (i.e. don't read in pneumonia case-crossover and mi case crossover). It will make the script run faster and won't confuse people reviewing your code.